common/reviews/api/node-core-library.api.md

```diff
@@ -7,7 +7,9 @@
 /// <reference types="node" />
 
 import * as child_process from 'child_process';
-import * as fs from 'fs';
+import * as fs from 'node:fs';
+import * as fs_2 from 'fs';
```

Comment:
This feels wrong

Reply:
Ah, guess I'll revert `node:fs` to `fs` in this file for now.